src/main/java/com/yo1000/sonarqube/collaboration/FizzBuzz.java

```diff
@@ -13,6 +13,12 @@ public String buzz(int value) {
     }
 
     public String fizzBuzz(int value) {
+        if (true) {
+            if (false) {
+                int a = 1;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "a" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://dev.yo1000.com/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "a". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://dev.yo1000.com/coding_rules#rule_key=squid%3AS1854)